### ISSUE TITLE
Why is staircase set to true in learning_rate?

### ISSUE THREAD

**Issue by kxhit** [charlesq34/pointnet#146]
I am curious of why staircase is set to true in learning_rate? Why not make the learning rate smoother? Thanks for comment!!!
